surround/experiment/web/experiment_explorer.html

```diff
@@ -76,6 +76,30 @@ <h1 class="title">Experiment Explorer</h1>
                     {% end %}
                 </tbody>
             </table>
+            {% if len(pages) > 1 %}
+                <hr class="hr" />
+                <nav class="pagination is-centered" role="navigation" aria-label="pagination">
+                    {% if page - 1 > 0 %}
+                        <a class="pagination-previous" href="./experiment?project_name={{project['project_name']}}&page={{page - 1}}">Previous</a>
+                    {% end %}
+                    {% if page + 1 <= pages[-1] %}
+                        <a class="pagination-next" href="./experiment?project_name={{project['project_name']}}&page={{page + 1}}">Next page</a>
+                    {% end %}
+                    <ul class="pagination-list">
+                        <li><a class="pagination-link" aria-label="Goto page {{pages[0]}}" href="./experiment?project_name={{project['project_name']}}">{{pages[0]}}</a></li>
```

Comment:
Is it possible to make this and the last page navigation not duplicating the list of pages in the middle?
For example, 
Instead of showing ```[1]...[1][2][3]...[5]```, show ```[1]...[2][3][4]...[7]```

Reply:
Done 👍